html/elements/link.json

```diff
@@ -146,19 +146,23 @@
           "__compat": {
             "support": {
               "chrome": {
-                "version_added": false
+                "version_added": true,
+                "notes": "Chrome does not include stylesheets which have the HTML <code>disabled</code> attribute in <code>document.styleSheets</code>."
               },
               "chrome_android": {
-                "version_added": false
+                "version_added": true,
+                "notes": "Chrome does not include stylesheets which have the HTML <code>disabled</code> attribute in <code>document.styleSheets</code>."
               },
               "edge": {
                 "version_added": true
               },
               "firefox": {
-                "version_added": false
+                "version_added": "68",
+                "notes": "Prior to Firefox 68, the HTML <code>disabled</code> attribute worked but could not be changed using JavaScript code."
```

Comment:
This is also wrong. It didn't work either way, the `disabled` element attribute did nothing. The `disabled` WebIDL attribute just reflected the `disabled` state of the stylesheet, if any.

Reply:
You're right -- I misread my test code's output. I'll remove the notes here.